GenumKit/Parsers/AssetsCatalogParser.swift

```diff
@@ -12,7 +12,8 @@ public final class AssetsCatalogParser {
 
   public init() {}
 
-  public func addImageName(name: String) -> Bool {
+  @discardableResult
+    public func addImage(named name: String) -> Bool {
```

Comment:
Huh, why the indentation change here? Seems like a misindentation was introduced by mistake